js/shared/record.js

```diff
@@ -188,13 +198,27 @@ window.ScratchpadRecord = Backbone.Model.extend({
             this.cacheRestore(-1 * this.seekCacheInterval);
         }
 
-        // Execute commands and build cache, bringing state up to current
-        for (var i = cacheOffset; i <= seekPos; i++) {
-            this.runCommand(this.commands[i]);
-            this.cache(i);
+        let currentOffset = cacheOffset;
+        const buildCache = () => {
+            const animationStep = 100;
+            const steps = Math.min(animationStep, seekPos - currentOffset + 1);
+            const newOffset = currentOffset + steps;
+
+            // console.log(`${currentOffset} / ${newOffset} / ${seekPos}`);
```

Comment:
Is this intended to be still in the code?

Reply:
Oops, no! Sorry about that. I'll remove it.